src/Symfony/Component/Console/Command/Command.php

```diff
@@ -684,6 +684,20 @@ public function addUsage(string $usage): static
         return $this;
     }
 
+    /**
+     * Adds a set of command usage examples.
+     *
+     * @param array[string] $usages
```

Comment:
```suggestion
     * @param string[] $usages
```

usually in the sf codebase, phpdoc is like so

Reply:
thanks!